src/Minimization_Monitor/image_gen/ush/MinMon_Plt.sh

```diff
@@ -2,26 +2,29 @@
 
 function usage {
   echo " "
-  echo "Usage:  MinMonPlt.sh MINMON_SUFFIX "
+  echo "Usage:  MinMonPlt.sh MINMON_SUFFIX [-p|--pdate -r|--run -t|--tank]"
   echo "            MINMON_SUFFIX is data source identifier that matches data in "
-  echo "              the $M_TANKverf/stats directory."
+  echo "              the $TANKDIR/stats or input tank directory."
```

Comment:
The `$` should be escaped with a `\` here so that "the $TANKDIR/stats or input tank directory" is printed out.  As is, "the /stats or input tank directory" is printed out, unless you happen to have $TANKDIR defined.

Reply:
Right you are.  Fixed.